pghoard/pghoard.py

```diff
@@ -126,14 +136,26 @@ def create_basebackup(self, cluster, connection_string, basebackup_path, callbac
             "--dbname", connection_string,
             "--format", "tar",
             "--xlog",
-            "--pgdata", raw_basebackup_path,
             "--progress",
             "--label", "initial_base_backup",
             "--verbose",
             ]
-        thread = PGBaseBackup(command, raw_basebackup_path, self.compression_queue, callback_queue)
+        if self.config["backup_sites"][site].get("multiple_tablespaces") is False:
```

Comment:
new configuration key is not documented anywhere, but it seems to affect backup behavior